Detectors/MUON/MCH/Contour/include/MCHContour/SVGWriter.h

```diff
@@ -72,7 +72,7 @@ class SVGWriter
   template <typename T>
   void polygon(const o2::mch::contour::Polygon<T>& p)
   {
-    mSVGBuffer << R"(<polygon points=")";
+    mSVGBuffer << R"(<polygon fill="blue" points=")";
```

Comment:
I guess this is a leftover from a test. Please remove it, as there's no point in hard coding a color here.